File: problems/1619-mean-of-array-after-removing-some-elements/solution.java
import java.util.Arrays;

class Solution {
    public double trimMean(int[] arr) {
        Arrays.sort(arr);
        int n = arr.length;
        int trim = n / 20;
        double sum = 0;
        for (int i = trim; i < n - trim; i++) {
            sum += arr[i];
        }
        return sum / (n - 2 * trim);
    }
}

File: problems/1619-mean-of-array-after-removing-some-elements/solution.py
class Solution:
    def trimMean(self, arr: List[int]) -> float:
        arr.sort()
        n = len(arr)
        remove_count = int(n * 0.05)
        trimmed_arr = arr[remove_count:n - remove_count]
        return sum(trimmed_arr) / len(trimmed_arr)